### ISSUE TITLE
Center label not getting rendered in React Donut chart

### ISSUE THREAD

**Issue by kalai001** [carbon-design-system/carbon-charts#175]
#### I am submitting a...

- [ ] Feature request
- [ ] Design defect
- [x] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:
0.9.0

### Issue description
Even with `DonutCenter` object passed, the Donut chart doesn't render the label.
Refer sandbox: https://codesandbox.io/s/k52rw2866r

### Screenshot or recording
![image](https://user-images.githubusercontent.com/46040341/52143555-e2d8b380-2681-11e9-9a27-66afd502828b.png)

**Comment by theiliad** [comment]
We've stopped using `DonutCenter` in chart configs for a while now. I fixed your codesandbox demo by removing `new DonutCenter` and just keeping the object you're using.

![image](https://user-images.githubusercontent.com/14989804/52157825-389c6400-2660-11e9-99a0-d72bb09c41ea.png)

The purpose of `DonutCenter` was for you to be able to manipulate the center after rendering the chart, separate than the rest of the donut. But currently we're debating whether that's a good idea. `DonutCenter` might be removed in v1.0 but the bottomline is that you shouldn't initialize your chart using it whether it goes or stays.
